### PR TITLE
Disentangle docsrs and features

### PR DESCRIPTION
**Pull request by pentamassiv**
Remaining cleanup from https://github.com/gtk-rs/gtk-rs-core/pull/1078

### CODE REVIEW

**Review by bilelmoussaoui — APPROVED** [review]

**Comment by bilelmoussaoui** [comment]
Now that the gir changes have landed, it would be great to also update & regenerate the sys crate.

**Comment by pentamassiv** [comment]
Are you asking me to do it and add it to the PR? I see that you approved it already, so I am not sure

**Comment by bilelmoussaoui** [comment]
> Are you asking me to do it and add it to the PR? I see that you approved it already, so I am not sure

Yes, please

Just make sure the update submodule commit is separate from the regeneration, as that would simplify backporting

**Comment by bilelmoussaoui** [comment]
Actually, I will do that along with more changes

File: gio/src/lib.rs [hunk 1]
```diff
@@ -84,19 +84,19 @@ pub use socket_msg_flags::SocketMsgFlags;
 mod subprocess;
 mod subprocess_launcher;
 mod threaded_socket_service;
-#[cfg(any(unix, docsrs))]
+#[cfg(unix)]
```
Comment (sdroege) [inline gio/src/lib.rs:87]: Is this actually correct? This way we won't get docs for this unless building on a unix-y system?

Comment (bilelmoussaoui) [reply gio/src/lib.rs:87]: I think that is correct yes, but i would refer to @GuillaumeGomez here

Comment (sdroege) [reply gio/src/lib.rs:87]: That seems like a problem then. Previously we got platform-specific docs when building docs on any platform. There's quite a bit of compat hackery for that :)

**Pull request merged**